server/app.py

```diff
@@ -0,0 +1,96 @@
+from flask import Flask, jsonify, request
+from flask_cors import CORS
+from datetime import datetime
+import json
+import logging 
+import os
+from dotenv import load_dotenv
+import requests
+
+# load environmental variable
+load_dotenv()
+API_KEY = os.getenv("API_KEY")
+Secret_KEY = os.getenv("Secret_KEY")
+
+app = Flask(__name__)
+allowed_origins = ["http://localhost:3000", "https://localhost:3000"]  
+logging.basicConfig(level=logging.DEBUG)
+CORS(app, origins=allowed_origins)  
+
+def callLLM(prompt):
+    def get_access_token():
+ 
+        url = f"https://aip.baidubce.com/oauth/2.0/token?grant_type=client_credentials&client_id={API_KEY}&client_secret={Secret_KEY}"
```

Comment:
Just curious, is it possible to pass `locale` to the endpoint such that it can return both simplified Chinese, traditional Chinese and English responses? Of course, the API endpoint needs to support it.

Reply:
can't find this function in the api, currently just modified the prompt to force its response into traditional chinese.